### ISSUE TITLE
New email address option for "Affiliate manager"

### ISSUE THREAD

**Issue by sumobi** [AffiliateWP/AffiliateWP#2214]
It might be worth exploring the possibility of adding a new "Affiliate manager email address" (or similar) input field in the affwp emails tab. I'd imagine some businesses like ours have a dedicated affiliate manager that isn't necessarily the site administrator.

All email notifications related to AffiliateWP would be sent there. Currently there's only 1 email, though we do get support tickets from time to time asking how to change this. It's only possible via a filter. Direct Link Tracking now sends an email to the admin when a new direct link has been submitted, so it would be good if any add-on that sends an admin email could go to this new address. If left blank it would simply fallback to the admin email as per usual.

**Comment by puternutt** [comment]
I completely agree, My client asked the same thing today, and so far there is no such field in current version 2.1.5. Any idea as to when this may be added or if there is a function code snippet that we can use to add this now?

**Comment by amdrew** [comment]
@puternutt We don't currently have a time frame for this. 

This code snippet will change the admin email option for new affiliate applications:

https://github.com/AffiliateWP/affiliatewp-code-snippet-library/blob/master/emails/change-affiliate-application-email.php

**Comment by gspice** [comment]
Another request:
https://secure.helpscout.net/conversation/444377678/68029/?folderId=205524

**Comment by gspice** [comment]
Another request to use something other than the admin email for new affiliate registrations:
https://secure.helpscout.net/conversation/598394904/83901?folderId=205529

**Comment by gspice** [comment]
Wants to send all registration emails to emails other than the site admin:
https://secure.helpscout.net/conversation/620309017/85951?folderId=1170842

**Comment by gspice** [comment]
Wants to change all admin email notifications (for referrals created and affiliate registration activity) to an email address other than admin.   
https://secure.helpscout.net/conversation/624263469/86312?folderId=1170842

**Comment by gspice** [comment]
I envision this on the _Settings > Email_ screen as a new line under each admin option where they can enter multiple email addresses separated by commas.  This way they can uncheck the admin options if they want to _only_ send to those alternate addresses.

![email-notifications](https://user-images.githubusercontent.com/3018245/43013915-4cc93b20-8bff-11e8-99a8-3b15e61536fc.png)
